docs/extensions/fares-v2.md

```diff
@@ -46,30 +46,19 @@ Items currently in discussion include:
-| Discussion         | [Cross Dataset Namespaces](https://docs.google.com/document/d/1AtaiD_RClXYGbo5oBJBlHSoFJea2FQ3nZeQlX1WKTdA/edit)                                | Ito World                        | None specific to proposal — impacts concepts outside of fares in GTFS | It’s impractical to merge all interrelated transit systems into 1 dataset. We need some way to represent relationships between different GTFS feeds.                                                              | Add namespaces.txt to define what external datasets are referred to within the feed.                                                                                         |
-| Discussion         | [GTFS-References](http://bit.ly/gtfs-references)                                                                                                | Elizabeth Sall, Cal-ITP          | fare_transfer_rules, transfers, stop_times, pathways, stops           | Same as the Cross Dataset Namespaces proposal. We need some way to represent relationships between different GTFS feeds.                                                                                          | Use references to identify when an entity (like stop, pathway, transfer, fare_transfer_rule) are shared across datasets.                                                     |
-| Discussion         | [fare_leg_join_rules.txt Proposal + Thoughts on “as_route”](https://bit.ly/gtfs-fares-v2-fare-leg-join-rules-proposal)                          | Brian Ferris, Google             | fare_leg_rules                                                        | The as_route field proposed in the full Fares V2 proposal does not sufficiently support complex multi-leg fare behaviors.                                                                                         | Add fare_leg_join_rules.txt as a more general mechanism for indicating when multiple consecutive legs should be considered as one for the purposes of fare calculation.      |
-| Discussion         | [Should fare transfers require that the same fare product is used on both legs?](https://docs.google.com/document/d/1zF5i03BFQi1kVBfN17aR7N1KRrpsHs3B9YjCzE8apJE/edit)                                                              | Nicholas Paun, Transit           | fare_leg_rules, fare_products                                         | It is currently ambiguous in the spec if the same fare_product must be used on both legs of a fare_transfer_rule in order for the transfer to be accepted.                                                        | Add 1 field to clearly define if the transfer requires the same fare_product or not.                                                                                         |
+<iframe src="https://portal.productboard.com/rhk8dbtic1iqakfznucry448" frameborder="0" width="100%", style="min-height:600px;"></iframe>
```

Comment:
This is great! 
Just to make sure I understand the purpose here: this is quick access to all the proposals related to GTFS-fares v2, right?

If so, where is the best place to see the [prioritized backlog](https://github.com/google/transit/issues/341#issuecomment-1339947915)? The next item we will be working on is the Variable pricing feature. 
It's almost like we need a "triage" column to put all the proposals we receive, and then we put them in the backlog based on when we want to work on it.

Reply:
Right now that information is in [our roadmap on the website](https://mobilitydata.org/roadmaps). We could merge all this info into the main portal and just add a "spec proposal triage" column so everything is in 1 place.